app/mailers/hearing_mailer.rb

```diff
@@ -120,9 +150,31 @@ def confirmation_subject
     end
   end
 
+  def virtual_conversion_confirmation_subject
```

Comment:
I think i prefer this naming better
```suggestion
  def convert_to_virtual_confirmation_subject
```

Reply:
Renamed this as suggested and renamed `not_virtual_conversion_confirmation_subject` to `convert_from_virtual_confirmation_subject` to match other naming changes.